merkletree/pad.go

```diff
@@ -19,27 +18,29 @@ var (
 // PAD is an acronym for persistent authenticated dictionary
 type PAD struct {
 	signKey      sign.PrivateKey
+	vrfKey       vrf.PrivateKey
 	tree         *MerkleTree // will be used to create the next STR
 	snapshots    map[uint64]*SignedTreeRoot
 	loadedEpochs []uint64 // slice of epochs in snapshots
 	latestSTR    *SignedTreeRoot
-	policies     Policies // the current policies in place
+	policies     []byte // the current policies in place
 }
 
 // NewPAD creates new PAD consisting of an array of hash chain
 // indexed by the epoch and its maximum length is len.
-func NewPAD(policies *Policies, signKey sign.PrivateKey, len uint64) (*PAD, error) {
+func NewPAD(policies []byte, signKey sign.PrivateKey, vrfKey vrf.PrivateKey, len uint64) (*PAD, error) {
 	if policies == nil {
 		panic("[merkletree] PAD must be created with a non-NULL Policies struct")
```

Comment:
Probably want to update this panic message to say "non-NULL policies", or something like that.